app/assets/javascripts/sequence_show.js

```diff
@@ -42,6 +45,26 @@ function init_sequence_show(data) {
         });
     }
 
+    function initD3TreeView(data, page) {
```

Comment:
jep


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3469) by @bmesuere on Mon Nov 09 2015 at 22:07._